app/package.json

```diff
@@ -14,7 +14,8 @@
     "package": "npm run copy-icons && npm run compile-electron && electron-builder build --publish never",
     "dev": "npm run compile-electron && cross-env ELECTRON_DEV=1 electron .",
     "dev-only-app": "npm run compile-electron && cross-env ELECTRON_DEV=1 ELECTRON_START_URL=http://localhost:3000 EXTERNAL_SERVER=true electron .",
-    "i18n": "npx --no-install i18next ./electron/main.ts -c ./electron/i18next-parser.config.js"
+    "i18n": "npx --no-install i18next ./electron/main.ts -c ./electron/i18next-parser.config.js",
+    "test": "jest"
```

Comment:
Did you forget to include jest in package.json? I see it only in the lock file.

Reply:
My clumsy clumsy fingers! Thanks.